src/app/global.css

```
@@ -762,24 +763,31 @@ button,
-  table-layout: fixed !important;
+  display: table-row !important;
+  width: auto !important;
+  table-layout: auto !important;
 }
 
 .prose table th,
 .prose table td {
-  word-wrap: break-word !important;
-  word-break: break-word !important;
-  overflow-wrap: break-word !important;
+  word-wrap: normal !important;
```

Stylelint (17.12.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)